external-docs/docs/guides/configuration/qube-config/columns/observations.md

```diff
@@ -0,0 +1,92 @@
+# Observations columns
+
+This page discusses what an observations column is, where one should be used, and how one can be defined.
+
+> For a detailed look at an observations column's configuration options, see the [Reference table](#reference) at the bottom of this page.
+
+## What is an observations column?
+
+Observations columns contain the numerical values of observations recorded in the data set, and are the most important component of a CSV-W. In order to be valid, a data cube must include at least one observations column, each of which must have a unit and a measure associated with it. Measures and units can either be defined against the observations column, or can be contained in separate [unit](./units.md) and [measure](./measures.md) columns.
```

Comment:
They're one of the most important components of a cube, CSV-Ws in general can have any kind of non-cube data in them.